cleanlab/experimental/label_issues_batched.py

```diff
@@ -446,6 +468,37 @@ def find_label_issues_batched(
     issue_indices : np.ndarray
       Indices of examples with label issues, sorted by label quality score.
     """
+    if labels_file is not None:
+        if labels is not None:
```

Comment:
Should we also raise value error if `labels` and `labels_file` are both `None`? Or is that self explanatory?

Reply:
Same with `pred_probs`

Reply:
it will raise an error already

Reply:
but I've made the error message more clear

Reply:
You mean in `assert labels is not None`? I was wondering if we should raise a more verbose error explaining one of the two needs to be passed in (since technically both the parameters are optional)

Reply:
ya have added one later in the code:

```
if labels is None:
        raise ValueError("must provide one of: `labels` or `labels_file`")
    if pred_probs is None:
        raise ValueError("must provide one of: `pred_probs` or `pred_probs_file`")
```